packages/fluentui/docs/src/examples/components/Button/Variations/ButtonExampleSize.shorthand.tsx

```diff
@@ -4,20 +4,12 @@ import * as React from 'react';
 const ButtonExampleSizeShorthand: React.FC = () => (
   <>
     <Flex gap="gap.smaller">
-      <Button size="smallest" content="Smallest" />
```

Comment:
I don't think this is a good change. Even though Teams theme only has different styling for `small`, the Fluent UI API still accepts all the different `size` values:
https://github.com/microsoft/fluentui/blob/c5b991e4e6b1264d2d9d3dae3e1725b0e50accf0/packages/fluentui/react-northstar/src/utils/commonPropTypes.ts#L23

We should keep those in the docsite as it is not Teams-specific.

Reply:
Ah, we should probably remove those too, then?

@codepretty

Reply:
I think that we should also update `ButtonProps` to use only supported sizes:

```tsx
size?: 'small' | 'medium'
```

No one really uses N* with different themes and I think that it's a way to improve UX as having unsupported things is confusing for users.

Reply:
@notandrew we had an offline conversation within the team and agreed to go with these changes.

Let's update `ButtonProps` to use only supported sizes:

```tsx
size?: 'small' | 'medium'
```

Reply:
@layershifter  are we skipping the deprecation part here ? Let's remove from the docs and deprecate the size.... just thinking about a good way to do this TS rn...

Reply:
@ling1726 we are skipping there deprecation because these values have never worked in Teams theme. TBH I am considering this as a fix rather than a breaking change.